docs/advanced/query_configuration.mdx

```diff
@@ -65,6 +65,8 @@ _coming soon_
 |top_p|Controls the diversity of words. Higher values (closer to 1) make word selection more diverse, lower values make words less diverse.|float|1|
 |history|include conversation history from your client or database.|any (recommendation: list[str])|None|
 |stream|control if response is streamed back to the user.|bool|False|
+|deployment_name|t.b.a.|str|None|
+|system_prompt|System prompt string. Unused if none.|str|None|
```

Comment:
In the docs, can you add an example of how to use this actually?

Reply:
added example: https://github.com/embedchain/embedchain/pull/448/files#diff-ceb7311e70d50f7fa3a14f5e727a03781b2a4c20c3e8105f3459d1d8af8f1160R71